public/app/features/explore/Logs/LogsMetaRow.tsx

```diff
@@ -1,21 +1,26 @@
 import { css } from '@emotion/css';
 import { memo } from 'react';
 
-import { LogsDedupStrategy, LogsMetaItem, LogsMetaKind, LogRowModel, CoreApp, Labels } from '@grafana/data';
+import { LogsDedupStrategy, LogsMetaItem, LogsMetaKind, LogRowModel, CoreApp, Labels, store } from '@grafana/data';
 import { config, reportInteraction } from '@grafana/runtime';
-import { Button, Dropdown, Menu, ToolbarButton, Tooltip, useStyles2 } from '@grafana/ui';
+import { Button, Dropdown, Menu, ToolbarButton, useStyles2 } from '@grafana/ui';
 import { Trans } from 'app/core/internationalization';
 
-import { LogLabels, LogLabelsList } from '../../logs/components/LogLabels';
-import { MAX_CHARACTERS } from '../../logs/components/LogRowMessage';
+import { LogLabels, LogLabelsList, Props as LogLabelsProps } from '../../logs/components/LogLabels';
 import { DownloadFormat, downloadLogs } from '../../logs/utils';
 import { MetaInfoText, MetaItemProps } from '../MetaInfoText';
 
+import { SETTINGS_KEYS } from './utils/logs';
+
 const getStyles = () => ({
   metaContainer: css({
     flex: 1,
     display: 'flex',
     flexWrap: 'wrap',
+    '& span': {
```

Comment:
Changing font weight would require to make the change in MetaInfoText, component that is used in other places.
Line height is also being set to properly align meta rows, otherwise key and values would be misaligned.